car/car_offset_writer.go

```diff
@@ -106,8 +106,13 @@ func (s *CarOffsetWriter) writeBlocks(ctx context.Context, w io.Writer, headerSi
 			return nil, fmt.Errorf("getting block %s: %w", c, err)
 		}
 
+		// take a copy of the links array before nd.RawData() triggers a sort
```

Comment:
the real solution is to get https://github.com/ipld/go-car/pull/291 to replace this class, right?

Reply:
yes, that's the proper thing that needs to be done